chromadb/test/property/invariants.py

```diff
@@ -75,46 +75,80 @@ def no_duplicates(collection: Collection):
     assert len(ids) == len(set(ids))
 
 
+def _exact_distances(
+    query: types.Embeddings,
+    targets: types.Embeddings,
+    distance_fn: Callable = lambda x, y: np.linalg.norm(x - y) ** 2,
```

Comment:
Accommodates different index types / metrics

Reply:
Can we pull out the l2 norm since we will have other distance funcs.

Reply:
Will do so in the index tests.